boards/arm/nucleo_f030r8/pinmux.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2017 RnDity Sp. z o.o.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <pinmux.h>
+#include <sys_io.h>
+#include "pinmux/pinmux.h"
+
+#include "pinmux/stm32/pinmux_stm32.h"
+
+/* pin assignments for NUCLEO-F030R8 board */
+static const struct pin_config pinconf[] = {
+#ifdef CONFIG_UART_STM32_PORT_1
+	{STM32_PIN_PB6,  STM32F0_PINMUX_FUNC_PB6_USART1_TX},
+	{STM32_PIN_PB7, STM32F0_PINMUX_FUNC_PB7_USART1_RX},
+#endif	/* CONFIG_UART_STM32_PORT_1 */
```

Comment:
Not in line with doc and device tree

Reply:
Done, aligned Doc, Pinmux and DT for USART1